src/AppBundle/DataFixtures/ORM/PostFixtures.php

```diff
@@ -184,9 +183,9 @@ private function getPhrases()
         ];
     }
 
-    private function getRandomPostTiles()
+    public static function getRandomPostTiles()
```

Comment:
@yceruto,  there is a typo in the method's name here. Should be `getRandomPostTitles`.